_data/navigation.yml

```diff
@@ -718,10 +720,12 @@ community:
   - title: Captain's Log
     url: /community/captains-log/
 
-publications:
-  - title: eBook
-    url: /publications/ebook/
-  - title: Presentations and Videos
-    url: /publications/presentations/
-  - title: Articles
-    url: /publications/articles/
+news:
+  - title: Sign Up
+    url: /news/signup
+  - title: Latest
+    url: /news/latest/
+  - title: Issues
```

Comment:
Maybe call it something other than Issues? My brain goes to github issues, not an issue of a newsletter/magazine/whatever

Reply:
volumes I guess?

Reply:
Editions?